src/geometry/planar.md

```diff
@@ -13,7 +13,7 @@ In this article we will deal with finding both inner and outer faces of a planar
 
 ## Some facts about planar graphs
 
-In this section we present several facts about planar graphs without proof. Readers who are interested in proofs should refer to [Graph Theory by R. Diestel](https://sites.math.washington.edu/~billey/classes/562.winter.2018/articles/GraphTheory.pdf) or some other book.
+In this section we present several facts about planar graphs without proof. Readers who are interested in proofs should refer to [Graph Theory by R. Diestel](https://diestel-graph-theory.com/) (see [video lectures](https://www.youtube.com/@DiestelGraphTheory) based on this book) or some other book.
```

Comment:
Should we link [here](https://www.math.uni-hamburg.de/home/diestel/books/graph.theory/preview/Ch4.pdf)? The link is from the [book site](https://diestel-graph-theory.com/basic.html) as well.

Reply:
Good point, changed the link.